app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new
+  end
```

Comment:
Use single line syntax for empty actions
```suggestion
  def new; end
```